src/news-app.html

```diff
@@ -247,10 +246,10 @@
         Polymer.AppLayout.scroll({ top: 0, behavior: 'silent' });
         // Close the drawer - in case the *route* change came from a link in the drawer.
         this.$.nav.closeDrawer();
-      },
+      }
 
-      _pageChanged: function(page, oldPage) {
-        var href;
+      _pageChanged(page, oldPage) {
+        let href;
 
         switch(page) {
```

Comment:
:tada: Great News! :tada: You can use `lazy-imports` here to do a declarative lazy import by a "group" name with whatever href you want.